cryptofeed/exchange/ftx.py

```diff
@@ -44,8 +45,28 @@ def _parse_symbol_data(cls, data: dict, symbol_separator: str) -> Tuple[Dict, Di
             symbol = d['name']
             ret[normalized] = symbol
             info['tick_size'][normalized] = d['priceIncrement']
+            
+            # Add index symbol for non-spot markets
+            if d['underlying'] is None or d['type'] == 'spot':
+                continue
+            index_normalized = cls._translate_index_symbol(d['underlying'], False)
+            ret[index_normalized] = index_normalized
+            info['index_to_derivative'][index_normalized] = symbol
+            info['is_index'][index_normalized] = True
```

Comment:
more comments here probably needed or we'll forget why this workaround is needed!